<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pB_OpenJson">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open JSON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_SaveJson">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save JSON</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sB_value1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dSB_value2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE_value3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cB_value4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>284</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
